packages/core/types/src/cart/workflows.ts

```diff
@@ -93,9 +93,6 @@ export interface UpdateCartWorkflowInputDTO {
 
 export interface CreatePaymentCollectionForCartWorkflowInputDTO {
   cart_id: string
-  region_id: string
```

Comment:
note: you'd also have to update the starter to account for these

Reply:
just had to adjust the `js-sdk` here.